### PR TITLE
Feature/token balances dao

### PR DESCRIPTION
**Pull request by yukamend**
still not fully finished, but functionality is there

### CODE REVIEW

**Comment by mohamedalichelbi** [comment]
TODOs:
- [ ] Display token Icons left to Token symbols and in circular form, similar to NEAR wallet.
- [ ] Move Tokens whitelist back next to Admins list.
- [ ] Load tokens when tabs are switched to view funds.
- [ ] Show the non-selected tabs in a clearer way (now they just blend in with the background).

File: src/near-config.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function getConfig(env) {
                 CRONCAT_MANAGER_ADDRESS: "manager_v1.croncat.near",
                 nodeUrl: "https://rpc.mainnet.near.org",
                 walletUrl: "https://wallet.near.org",
-                helperUrl: "https://helper.mainnet.near.org",
```
Comment (lennczar) [inline src/near-config.js:14]: whats this?

File: src/shared/ui/components/table/index.tsx [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+} from "@mui/material";
+
+// import { visuallyHidden } from '@mui/utils'
+import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
+import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
+import React, { useState } from "react";
+
+import { useBreakpoint } from "../../lib/breakpoints";
+
+const StyledTableContainer = styled(TableContainer)({
```
Comment (lennczar) [inline src/shared/ui/components/table/index.tsx:22]: Please move all css to a seperate file

File: src/shared/ui/components/table/index.tsx [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+                                                onClick={createSortfunction(string)}
+                                                IconComponent={sortIcon}
+                                                sx={{
+                                                    "& .MuiTableSortLabel-icon": {
+                                                        transform: "none",
+                                                        opacity: 1,
+                                                    },
+
+                                                    "& .MuiTableSortLabel-iconDirectionDesc .sort-down": {
+                                                        fill: (theme) =>
```
Comment (lennczar) [inline src/shared/ui/components/table/index.tsx:273]: for css like this lets find a different solutions, we are not even using an mui theme

File: src/shared/ui/components/tabs/index.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    const [activeТаbIndex, activeTabSwitch] = React.useState(0);
+
+    const onChange = useCallback(
+        (_event: React.ChangeEvent<any>, value: any) => activeTabSwitch(value),
+        [activeTabSwitch]
+    );
+
+    return (
+        <Box sx={{ width: "100%" }}>
+            <Box>
```
Comment (lennczar) [inline src/shared/ui/components/tabs/index.tsx:21]: unnecessary `<Box>`. Also please use divs. To my understanding the only benefit of Box is that is works with sx props. But we dont have/want CSS in JS.

File: src/shared/ui/components/token-label/token-label.scss [hunk 5]
```diff
@@ -0,0 +1,17 @@
+@use "sass/mixin";
+
+.token-label {
+    display: flex;
+    align-items: center;
+    gap: 14px;
```
Comment (lennczar) [inline src/shared/ui/components/token-label/token-label.scss:6]: please do not use hard coded values.

**Review by lennczar — CHANGES_REQUESTED** [review]
Please merge the two `components` directories

File: src/near-config.js (hunk 1 above)
Comment (Rin-Akaia-eth) [reply src/near-config.js:14]: @yukamend

Comment (Rin-Akaia-eth) [reply src/near-config.js:14]: /cast @mohamedalichelbi

**Review by mohamedalichelbi — APPROVED** [review]

**Review by lennczar — APPROVED** [review]

**Pull request merged**